{
  "Severity": "MODERATE",
  "UpdatedAt": "2023-09-05T14:05:20Z",
  "Package": {
    "Ecosystem": "PIP",
    "Name": "django"
  },
  "Advisory": {
    "DatabaseId": 2140,
    "Id": "MDE2OlNlY3VyaXR5QWR2aXNvcnlHSFNBLXdwanItajU3eC13eGZ3",
    "GhsaId": "GHSA-wpjr-j57x-wxfw",
    "References": [
      {
        "Url": "https://nvd.nist.gov/vuln/detail/CVE-2020-13254"
      },
      {
        "Url": "https://docs.djangoproject.com/en/3.0/releases/security/"
      },
      {
        "Url": "https://groups.google.com/d/msg/django-announce/pPEmb2ot4Fo/X-SMalYSBAAJ"
      },
      {
        "Url": "https://www.djangoproject.com/weblog/2020/jun/03/security-releases/"
      },
      {
        "Url": "https://lists.debian.org/debian-lts-announce/2020/06/msg00016.html"
      },
      {
        "Url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/4A2AP4T7RKPBCLTI2NNQG3T6MINDUUMZ/"
      },
      {
        "Url": "https://security.netapp.com/advisory/ntap-20200611-0002/"
      },
      {
        "Url": "https://usn.ubuntu.com/4381-1/"
      },
      {
        "Url": "https://usn.ubuntu.com/4381-2/"
      },
      {
        "Url": "https://www.debian.org/security/2020/dsa-4705"
      },
      {
        "Url": "https://www.oracle.com/security-alerts/cpujan2021.html"
      },
      {
        "Url": "https://github.com/django/django/commit/07e59caa02831c4569bbebb9eb773bdd9cb4b206"
      },
      {
        "Url": "https://github.com/django/django/commit/84b2da5552e100ae3294f564f6c862fef8d0e693"
      },
      {
        "Url": "https://github.com/advisories/GHSA-wpjr-j57x-wxfw"
      }
    ],
    "Identifiers": [
      {
        "Type": "GHSA",
        "Value": "GHSA-wpjr-j57x-wxfw"
      },
      {
        "Type": "CVE",
        "Value": "CVE-2020-13254"
      }
    ],
    "Description": "An issue was discovered in Django version 2.2 before 2.2.13 and 3.0 before 3.0.7. In cases where a memcached backend does not perform key validation, passing malformed cache keys could result in a key collision, and potential data leakage.",
    "Origin": "UNSPECIFIED",
    "PublishedAt": "2020-06-05T16:20:44Z",
    "Severity": "MODERATE",
    "Summary": "Data leakage via cache key collision in Django",
    "UpdatedAt": "2023-09-05T14:05:20Z",
    "WithdrawnAt": "",
    "CVSS": {
      "Score": 5.9,
      "VectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N"
    }
  },
  "Versions": [
    {
      "FirstPatchedVersion": {
        "Identifier": "3.0.7"
      },
      "VulnerableVersionRange": "\u003e= 3.0.0, \u003c 3.0.7"
    },
    {
      "FirstPatchedVersion": {
        "Identifier": "2.2.13"
      },
      "VulnerableVersionRange": "\u003e= 2.0.0, \u003c 2.2.13"
    }
  ]
}